### PR TITLE
122 Client Information Permissions

### PR DESCRIPTION
**Pull request by MatthewMeacham**
This adds in a permission for a volunteer to 'view all client information' meaning specifically a client's last name, email address, and phone number. 

**TESTING**
* Run the database scripts
* Not logged in, go to the public queue and ensure that you only see a client's first name and last initial
* Logged into preparer@test.test, go to the private queue and ensure that you can see only a client's first name and last initial. Check the public queue, and it should be the same case. You should also not be able to see a client's email or phone number anywhere.
* Logged into siteadmin@test.test, go to the private queue and ensure that you can see a client's first name, full last name, phone number, and email (when you click on a client of course)

**Randomly adding in the new Database ERD here because the wiki won't let me upload images there, only link them, so I am creating a link in this PR**
databaseerd

### CODE REVIEW

File: server/queue_priv.php [hunk 1]
```diff
@@ -97,29 +95,3 @@ function cancelledAppointment($id) {
 		echo json_encode($appointment);
 		$stmt = null;
 	}
-
```
Comment (MatthewMeacham) [inline server/queue_priv.php]: This all seemed like dead code from what I could tell, so I removed it.

File: queue/private.php [hunk 2]
```diff
@@ -34,14 +34,14 @@
 
 				<div class="queue" ng-if="appointments.length > 0" ng-cloak>
 					<div class="row queue-row py-1 pointer"
-							 ng-repeat="appointment in appointments | orderBy:'scheduledTime' | searchFor: clientSearch"
+							 ng-repeat="appointment in appointments | orderBy:['timeIn == null', 'timeReturnedPapers == null', 'timeAppointmentStarted == null', 'scheduledTime'] | searchFor: clientSearch"
```
Comment (CollinsSpencer) [inline queue/private.php]: I like this.

File: queue/private.php [hunk 3]
```diff
@@ -111,7 +116,7 @@
 						 	</form>
```
Comment (CollinsSpencer) [inline queue/private.php]: This form still has some bugs. For example, it doesn't clear when I switch between clients. Also, I can still type more than 255 characters into the field. The textarea has ng-maxlength="255" but it also needs maxlength="255". This issue doesn't match the scope of this ticket though. I just wanted to make note of it.

Comment (MatthewMeacham) [reply queue/private.php]: Created https://github.com/UNL-CSE-AMBASSADORS/VITA/issues/152

**Pull request merged**